contrib/epee/include/storages/portable_storage.h

```diff
@@ -31,6 +31,8 @@
 #include "misc_log_ex.h"
 #include "span.h"
 
+#include <boost/mpl/contains.hpp>
```

Comment:
It's OK for a fix, but will increase the compilation time, i.e. something that I'll have to handle later on.

Reply:
Suggest the better one now.

Reply:
(unless it's not an easy header swap ?)

Reply:
I would like to be able to tell this, but I can't do it with 100% certainty without this framework.

Reply:
There's `mp11` which is (compile-time) faster but requires a newer minimum boost version.

Reply:
Yes. It's better to sacrifice some compilation time for the time being, while allowing a wider range of boost versions.